tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -67,3 +69,76 @@ func TestAuthNPolicy(t *testing.T) {
 		}
 	})
 }
+
+func TestAuthNJwt(t *testing.T) {
+	// V1alpha3 == true implies envoyv2, jwt authn doesn't work for v2 so skip it now.
+	if tc.V1alpha3 {
+		t.Skipf("Skipping %s: V1alpha3=true", t.Name())
+	}
+
+	// JWT token used is borrowed from https://github.com/istio/proxy/blob/master/src/envoy/http/jwt_auth/sample/correct_jwt.
+	// The Token expires in year 2132, issuer is 628645741881-noabiu23f5a8m8ovd8ucv698lj78vv0l@developer.gserviceaccount.com.
+	// Test will fail if this service account is deleted.
+	p := "testdata/authn/v1alpha1/correct_jwt"
+	token, err := ioutil.ReadFile(p)
+	if err != nil {
+		t.Fatalf("failed to read %q", p)
+	}
+	validJwtToken := string(token)
+
+	cfgs := &deployableConfig{
+		Namespace:  tc.Kube.Namespace,
+		YamlFiles:  []string{"testdata/authn/v1alpha1/authn-policy-jwt.yaml.tmpl"},
+		kubeconfig: tc.Kube.KubeConfig,
+	}
+	if err := cfgs.Setup(); err != nil {
+		t.Fatal(err)
+	}
+	defer cfgs.Teardown()
+
+	cases := []struct {
+		dst          string
+		src          string
+		path         string
+		port         string
+		validToken   string
```

Comment:
Why do we need 2? Can it simply be just token?

Reply:
want some cases using random string as token.

Reply:
You don't need 2 variables. Just one and set to the string you want in test cases.

Reply:
done